packages/vite-plugin-tanstack-start/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@netlify/vite-plugin-tanstack-start",
+  "version": "0.0.0",
+  "description": "Vite plugin for TanStack Start on Netlify",
+  "type": "module",
+  "engines": {
+    "node": "^22.12.0"
+  },
+  "main": "./dist/main.js",
+  "exports": "./dist/main.js",
+  "types": "./dist/main.d.ts",
+  "files": [
+    "dist/**/*"
+  ],
+  "scripts": {
+    "build": "tsup-node",
+    "prepack": "npm run build",
+    "test": "vitest run",
+    "test:dev": "vitest",
+    "test:ci": "vitest run",
+    "dev": "tsup-node --watch",
+    "publint": "npx -y publint --strict"
+  },
+  "keywords": [
+    "netlify",
+    "tanstack",
+    "tanstack-start",
+    "vite-plugin",
+    "dev",
+    "build"
+  ],
+  "license": "MIT",
+  "repository": "netlify/primitives",
+  "bugs": {
+    "url": "https://github.com/netlify/primitives/issues"
+  },
+  "author": "Netlify Inc.",
+  "devDependencies": {
+    "@netlify/dev-utils": "^4.1.3",
+    "@types/node": "^22.18.5",
+    "normalize-package-data": "^8.0.0",
+    "playwright": "^1.52.0",
+    "tsup": "^8.0.0",
+    "vite": "^7.1.5",
+    "vitest": "^3.0.0"
+  },
+  "dependencies": {
+    "@netlify/dev": "^4.5.9",
+    "@netlify/vite-plugin": "^2.5.10"
+  },
+  "peerDependencies": {
+    "@tanstack/react-start": "alpha",
+    "@tanstack/solid-start": "alpha",
+    "vite": ">=7.0.0"
```

Comment:
this is going to be TanStack Start's supported range, so no use in supporting any greater